### PR TITLE
Automatic CMS Editor Links

### PR DESCRIPTION
**Pull request by bglw**
Adds `data-cms-editor-link="cloudcannon:#..."` to all components where possible.

### CODE REVIEW

File: javascript-modules/live/lib/app/live.js [hunk 1]
```diff
@@ -23,101 +25,52 @@ export const getLive = (engines) => class BookshopLive {
...
-        const iterator = document.evaluate("//comment()[contains(.,'bookshop-live')]", document, null, XPathResult.ANY_TYPE, null);
-        const stack = [];
-        const updates = [];
-        let bindings = {}
-        let currentNode = iterator.iterateNext();
-        while(currentNode){
-            const matches = currentNode.textContent.match(/bookshop-live ((?<end>end)|name\((?<name>[^)]*)\) params\((?<params>[^)]*)\)).*/);
-            const contextMatches = currentNode.textContent.match(/bookshop-live.*context\((?<context>.+)\)\s*$/);
+    async render(options = {}) {
+        const CCEditorPanelSupport = typeof window !== 'undefined' && window.CloudCannon?.refreshInterface;
```
Comment (bglw) [inline javascript-modules/live/lib/app/live.js:45]: @GeorgePhillips does this look good?

Comment (bglw) [inline javascript-modules/live/lib/app/live.js:45]: @GeorgePhillips does this look good?

Comment (GeorgePhillips) [reply javascript-modules/live/lib/app/live.js:45]: Looks good to me

**Comment by GeorgePhillips** [comment]
🙈 Looks good

**Review by GeorgePhillips — APPROVED** [review]

**Pull request merged**